common/reference/index-fastopt.html

```diff
@@ -0,0 +1,445 @@
+<!DOCTYPE html>
+<!-- Copyright 2013 the V8 project authors. Scroll to the end for full license -->
+<html lang="en">
+<head>
+    <meta http-equiv="X-UA-Compatible" value="IE=edge"/>
+    <meta charset="utf-8"/>
+    <title>Octane 2.0 JavaScript Benchmark</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="Content-Script-Type" content="text/javascript">
+    <meta name="description"
+          content="Octane 2.0 JavaScript Benchmark - the Javascript benchmark for the modern web">
+    <meta name="author" content="Octane Team Google">
+    <!-- twitter bootstrap code -->
+    <script src="js/jquery.js"></script>
+    <script src="js/bootstrap-transition.js"></script>
+    <script src="js/bootstrap-collapse.js"></script>
+    <!-- Octane benchmark code -->
+    <script type="text/javascript" src="base.js"></script>
+    <script type="text/javascript" src="../../richards/target/scala-2.11/richards-fastopt.js"></script>
+    <script type="text/javascript" src="../../richards/target/scala-2.11/richards-launcher.js"></script>
+    <script type="text/javascript" src="../../deltablue/target/scala-2.11/deltablue-fastopt.js"></script>
+    <script type="text/javascript" src="../../deltablue/target/scala-2.11/deltablue-launcher.js"></script>
+    <!--<script type="text/javascript" src="crypto.js"></script>-->
+    <script type="text/javascript" src="../../raytrace/target/scala-2.11/raytrace-fastopt.js"></script>
+    <script type="text/javascript" src="../../raytrace/target/scala-2.11/raytrace-launcher.js"></script>
+    <!--<script type="text/javascript" src="earley-boyer.js"></script>-->
+    <!--<script type="text/javascript" src="regexp.js"></script>-->
+    <!--<script type="text/javascript" src="splay.js"></script>-->
+    <!--<script type="text/javascript" src="navier-stokes.js"></script>-->
+    <!--<script type="text/javascript" src="pdfjs.js"></script>-->
+    <!--<script type="text/javascript" src="mandreel.js"></script>-->
+    <!--<script type="text/javascript" src="gbemu-part1.js"></script>-->
+    <!--<script type="text/javascript" src="gbemu-part2.js"></script>-->
+    <!--<script type="text/javascript" src="code-load.js"></script>-->
+    <!--<script type="text/javascript" src="box2d.js"></script>-->
+    <!--<script type="text/javascript" src="zlib.js"></script>-->
+    <!--<script type="text/javascript" src="zlib-data.js"></script>-->
+    <!--<script type="text/javascript" src="typescript.js"></script>-->
+    <!--<script type="text/javascript" src="typescript-input.js"></script>-->
+    <!--<script type="text/javascript" src="typescript-compiler.js"></script>-->
+
+    <script type="text/javascript">
+  var completed = 0;
+  var benchmarks = BenchmarkSuite.CountBenchmarks();
```

Comment:
I would still like to see a separate .js file for this script, factoring out stuff from the 3 HTML files.